src/Admin/AbstractAdmin.php

```diff
@@ -675,6 +675,7 @@ final public function getClass(): string
             }
 
             $subClass = $this->getRequest()->query->get('subclass');
+            \assert(\is_string($subClass));
```

Comment:
should we add a casting instead of assert when using `$request->query->get('x')`?

Reply:
There is already the check
```
final public function hasActiveSubClass(): bool
    {
        if (\count($this->subClasses) > 0 && $this->hasRequest()) {
            return \is_string($this->getRequest()->query->get('subclass'));
        }

        return false;
    }
```

So I'm sure it's a string.